charts/capsule/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 manager:
   image:
-    repository: quay.io/clastix/capsule
+    repository: clastix/capsule
     pullPolicy: IfNotPresent
     tag: ''
 
```

Comment:
We should also change kubectl [image registry](https://github.com/clastix/capsule/pull/575/files#diff-8facec49e536e52cd2044382bdb9bbd0227328f1977cfee6dc55f73886a67b7bR43), don't we ?

Reply:
Nice spot! Created the Docker Hub [repository](https://hub.docker.com/repository/docker/clastix/kubectl), copying in the meanwhile the images from Quay.io to Docker.io!